src/app/cart/cart.tpl.html

```diff
@@ -40,7 +40,7 @@ <h3 class="panel-name" translate>Your Gift Cart</h3>
                   <tr class="giftsum-gift-row" ng-repeat="i in $ctrl.cartData.items">
                     <td class="td-gift">
                       <img desig-src="{{i.designationNumber}}" class="giftsum-profile pull-left" width="90" height="51">
-                      <span class="giftsum-person giftsum-title"><a ng-href="/{{i.designationNumber}}">{{i.displayName}}</a></span>
+                      <span id="cart-{{$index}}" class="giftsum-person giftsum-title"><a ng-href="/{{i.designationNumber}}">{{i.displayName}}</a></span>
```

Comment:
If there is a desire to have something meaningful rather than an index, in the context of a cart, the designation number/frequency combination is unique.